mscore/scoreBrowser.cpp

```diff
@@ -298,5 +306,178 @@ void ScoreBrowser::setScoreActivated(QListWidgetItem* val)
       emit scoreActivated(item->info().filePath());
       }
 
+//---------------------------------------------------------
+//   focusNextPrevChild
+//---------------------------------------------------------
+
+bool ScoreBrowser::focusNextPrevChild(bool next)
```

Comment:
It's used to override the default tab behavior, so I can't remove the function. Without it, using tab interferes with template navigation.